src/main/resources/lang/default.lang

```diff
@@ -1253,6 +1253,16 @@ heal reasons:
 	wither: withering, withered, wither potion, a wither effect, wither effect, wither
 	wither_spawn: wither spawning, wither spawn, wither summoning, a wither summoning, a wither spawn, a wither spawning
 
+# -- Fishing States --
+fishing states:
```

Comment:
`fishing states` was not added to the `types:` section node. (place under #bukkit comment)